cves/kernel/CVE-2021-38166.yml.bak

```diff
@@ -473,5 +472,5 @@ nickname_instructions: |
   A catchy name for this vulnerability that would draw attention it.
   If the report mentions a nickname, use that.
   Must be under 30 characters. Optional.
-nickname:
+nickname: Hash bucket size int overflow
```

Comment:
Certainly describes what it is. The nickname is a bit long though, how about OBFlow for Over Bucket Flow? 😆